clickbait-generator.rb

```diff
@@ -4,9 +4,12 @@
 MESSAGES = YAML.load_file("./messages.yml")["messages"]
 
 class ClickbaitGenerator
-  def self.get (url, project_name, language_name)
+  def self.get (url, project_name, language_name, emoji)
     r = Random.new
     template = MESSAGES[r.rand(MESSAGES.count - 1)]
```

Comment:
think you could just do: `MESSAGES.sample` to grab a random message if thats indeed whats intended there?